### DISCUSSION TITLE
Access other docker container ports

### DISCUSSION THREAD

**Discussion by ThierryBegin** [dec0dOS/zero-ui#142] [Q&A] [answered]
I have plenty of other services running in docker instances on the same machine as the controller. unfortunately it seem that even when I'm connected to the node, I can access other machines different services but not the ones on the controller. 
not sure how to solve that.
```services:
  zerotier:
    image: zyclonite/zerotier:latest
    container_name: zu-controller
    restart: unless-stopped
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
    environment:
      - ZT_OVERRIDE_LOCAL_CONF=true
      - ZT_ALLOW_MANAGEMENT_FROM=0.0.0.0/0
    expose:
      - "9993"
    ports:
      - "9993:9993"
    cap_add:
      - NET_ADMIN
    devices:
      - /dev/net/tun
  zero-ui:
    image: dec0dos/zero-ui:latest
    container_name: zu-main
    build:
      context: .
      dockerfile: ./docker/zero-ui/Dockerfile
    restart: unless-stopped
    depends_on:
      - zerotier
    volumes:
      - ./zerotier-one:/var/lib/zerotier-one
      - ./data:/app/backend/data
    environment:
      - ZU_CONTROLLER_ENDPOINT=http://zerotier:9993/
      - ZU_SECURE_HEADERS=true
      - ZU_DEFAULT_USERNAME=admin
      - ZU_DEFAULT_PASSWORD=zero-ui
    expose:
      - "4000"
    ports:
      - "5100:4000"
```
![image](https://user-images.githubusercontent.com/2735823/216420208-f7c54fa2-7337-4b5a-ad91-082d5c2e456c.png)

**Comment by dec0dOS** [comment, marked as answer]
Hello, @ThierryBegin. You should use another ZeroTier instance on the host itself or just use one zerotier instance on the host also as a controller.

**Comment by ThierryBegin** [comment]
problem with that is that the port is already used so it conflicts

**Reply by dec0dOS** [reply]
You could use another port. Check the ZeroTier docs on how to do it.

> or just use one zerotier instance on the host also as a controller.
